### PR TITLE
For now i've changed some Ui with text color of login and signup and …

### PR DESCRIPTION
**Pull request by Nimitshekhawat**
…also want to explore further code if u merge this :)

### CODE REVIEW

**Comment by Nimitshekhawat** [comment]
please review it

File: lib/pages/login_page.dart [hunk 1]
```diff
@@ -158,7 +158,7 @@ class _LoginState extends State<Login> {
                       ),
                     ),
                   ),
```
Comment (Nimitshekhawat) [inline lib/pages/login_page.dart:160]: here i just add some text styling in it

File: lib/pages/login_page.dart [hunk 2]
```diff
@@ -176,7 +176,7 @@ class _LoginState extends State<Login> {
                       ),
                     ),
                   ),
-                  child: const Text("Sign Up"),
```
Comment (Nimitshekhawat) [inline lib/pages/login_page.dart:179]: same above i add some styleing

**Review by Nimitshekhawat — COMMENTED** [review]
I add some styling in login page ui , hope u like and it motivated me to dive deep into your code and made some big changes

**Comment by yagnik2411** [comment]
@Nimitshekhawat  i will review it in 1-2 days

**Review by Nimitshekhawat — COMMENTED** [review]
Here i change sign up button ui and also change the depreciated widget with new ones

**Review by Nimitshekhawat — COMMENTED** [review]
change some ui and also update some widget which was depriciated

**Comment by yagnik2411** [comment]
sorry for inconvenience, i will your PR before EOD of 7-10-24

**Review by yagnik2411 — APPROVED** [review]
@Nimitshekhawat  this is miner changes you can add major things

**Pull request merged**